src/llama-hparams.h

```diff
@@ -123,6 +123,7 @@ struct llama_hparams {
     llama_swa_type swa_type = LLAMA_SWA_TYPE_NONE;
     // the size of the sliding window (0 - no SWA)
     uint32_t n_swa = 0;
+    uint32_t n_swa_pattern = 1;
```

Comment:
NIT: I think naming this `n_` implies that it's a "how many" parameter, so I think this should maybe be `swa_period` or something like that?